test/cache_invalidation_test.rb

```diff
@@ -46,14 +46,14 @@ def test_reload_invalidate_cached_objects
   def test_after_a_reload_the_cache_perform_as_expected
     Item.cache_has_many :associated_records, :embed => :ids
 
-    assert_equal [@baz, @bar], @record.associated_records
     assert_equal [@baz, @bar], @record.fetch_associated_records
+    assert_equal [@baz, @bar], @record.associated_records
```

Comment:
Calling `@record.associated_records` first causes the active record association to be loaded, after which `@record.fetch_associated_records` will just return the active record association, which doesn't really test the cache reloading.

Actually, now that I think about it, I'm not sure why the `assert_equal [@baz, @bar], @record.associated_records` lines are needed at all.  We don't need to test active record here.